_posts/2020-05-15-issue-2.md

```diff
@@ -0,0 +1,71 @@
+
+Tim Bray, former VP and software engineer at Amazon Web Services, quit his job earlier this month in protest of Amazon's treatment of warehouse workers who have been organizing during the pandemic around the world. In his [blog post](https://www.tbray.org/ongoing/When/202x/2020/04/29/Leaving-Amazon) about the resignation, he writes: _"remaining an Amazon VP would have meant, in effect, signing off on actions I despised. So I resigned."_ We applaud Tim for his resolve, and we stand in solidarity with the workers at Amazon who are organizing against the company at all levels. As Tim says, _"Amazon is exceptionally well-managed and has demonstrated great skill at spotting opportunities and building repeatable processes for exploiting them. It has a corresponding lack of vision about the human costs of the relentless growth and accumulation of wealth and power."_
+
+[California is suing Uber and Lyft](https://www.nytimes.com/2020/05/05/technology/california-uber-lyft-lawsuit.html) for misclassifying drivers as independent contractors rather than employees under [AB-5](https://en.wikipedia.org/wiki/California_Assembly_Bill_5_(2019)), a new state law that took effect on January 1, 2020. This is a significant win for workers and organizing groups like [Rideshare Drivers United](https://drivers-united.org) who have been advocating for its implementation and enforcement. Rideshare drivers are tech workers, too. If you are a driver, [you can file a wage claim here](https://drivers-united.org/a/submit-a-wage-claim).
+
+In Chicago, Instacart shopper and organizer Matthew Telles of [Gig Workers Collective](https://www.gigworkerscollective.org) gives a personal look into the deteriorating working conditions that prompted gig workers to fight back in [an interview for Picture Theory](https://podcasts.apple.com/us/podcast/matthew-telles-gig-workers-are-standing-up-to-big-tech/id1361867735?i=1000473489342).
```

Comment:
Add more at the end of this paragraph to quote Matthew, and to match the size of the other paragraphs:

“Grocery workers are dying and Instacart isn’t doing them any favors,” says Matthew. He also talks about how Instacart customers lure shoppers with "tip baiting," how pay and pretty much everything bottom-out after the first 90 days of being a shopper, and the unique challenges of building solidarity among gig workers and independent contractors.

Reply:
done